tests/unit/synapseutils/unit_test_synapseutils_sync.py

```diff
@@ -408,8 +408,8 @@ def test_sync_from_synapse_download_file_is_false(syn: Synapse) -> None:
         new_callable=AsyncMock,
         side_effect=[
             mocked_project_rest_api_dict(),
-            mocked_folder_rest_api_dict(),
             mocked_file_rest_api_dict(),
+            mocked_folder_rest_api_dict(),
         ],
```

Comment:
Due to swapping over to using an AsyncIO queue the files are instantly downloaded as they're added to the queue, instead of waiting for the code to execute all of the tasks as Folder tasks are added to the list. It changes the order of operations slightly, and as a result - Broke these unit tests.